src/controller/python/chip/utils/CommissioningBuildingBlocks.py

```diff
@@ -62,7 +62,8 @@ async def GrantPrivilege(adminCtrl: ChipDeviceController, grantedCtrl: ChipDevic
     '''
     data = await adminCtrl.ReadAttribute(targetNodeId, [(Clusters.AccessControl.Attributes.Acl)])
     if 0 not in data:
-        raise ValueError("Did not get back any data (possible cause: controller has no access..")
+        raise ValueError(
```

Comment:
would you mind reverting the changes that are formatting only? it'll make the PR easier to parse.

It seems like we haven't been enforcing line length formatting in these files and you probably have an auto-formatter on. vscode?

Reply:
Yes, vscode automatically formats the code. I had to modify my current settings but I have reverted the changes.